fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -0,0 +1,149 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.statistic.Constants;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+
+public class Task implements Writable {
+
+    @SerializedName("id")
+    private Long id;
```

Comment:
long？

Reply:
Yes, this is the internal ID of the Task, which is used to uniquely mark the Task. There may be a rename operation of the Task in the future.

Reply:
+1

Reply:
use long not Long